src/main.rs

```diff
@@ -1,18 +1,19 @@
 mod config;
 use actix_web::{get, App, HttpServer, Responder};
 use anyhow::Result as AnyhowResult;
+use tracing_actix_web::TracingLogger;
 
 #[get("/")]
```

Comment:
this looks like a GET endpoint - if so, is it going to call the `root` method when its hit?

Reply:
Yes, if a client calls our server at `/` with a GET request, it will trigger this method

E.g. Given:
```
Host: localhost
Port: 8080
```

Calling `GET http://localhost:8080/` will invoke root, and actix-web will handle creating a response containing whatever `root` returns (In this case just `"I am root"`)